File: JAVA PROGRAMMING/Test10.java
// 10. QUESTIONS BASED ON RELATIONAL OPERATORS
import java.util.Scanner;
public class Test10 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Input: ");
        int input = in.nextInt();

        if(input >= 90) {
            System.out.println("Grade A");
        } else if (input >= 80 && input < 90) {
            System.out.println("Grade B");
        } else if(input >= 70 && input < 80) {
            System.out.println("Grade C");
        } else {
            System.out.println("Grade D");
        }

        in.close();
    }
}

File: JAVASCRIPT PROGRAMMING/Test10.js
"use strict";
// 10. QUESTIONS BASED ON RELATIONAL OPERATORS
const func = (mark) => {
    if (mark <= 100 && mark >= 90) {
        console.log("Grade A+");
    }
    else if (mark <= 89 && mark >= 80) {
        console.log("Grade A");
    }
    else if (mark <= 79 && mark >= 70) {
        console.log("Grade B");
    }
    else if (mark <= 69 && mark >= 60) {
        console.log("Grade C");
    }
    else {
        console.log("Grade D");
    }
};
func(58);
func(65);
func(72);
func(88);
func(92);